include/core/jbackend.h

```diff
@@ -61,79 +66,365 @@ struct JBackend
+//object backend ->
+gboolean j_backend_object_init(JBackend*, gchar const*);
+void j_backend_object_fini(JBackend*);
 
-gboolean j_backend_object_init (JBackend*, gchar const*);
-void j_backend_object_fini (JBackend*);
+gboolean j_backend_object_create(JBackend*, gchar const*, gchar const*, gpointer*);
+gboolean j_backend_object_open(JBackend*, gchar const*, gchar const*, gpointer*);
 
-gboolean j_backend_object_create (JBackend*, gchar const*, gchar const*, gpointer*);
-gboolean j_backend_object_open (JBackend*, gchar const*, gchar const*, gpointer*);
+gboolean j_backend_object_delete(JBackend*, gpointer);
+gboolean j_backend_object_close(JBackend*, gpointer);
 
-gboolean j_backend_object_delete (JBackend*, gpointer);
-gboolean j_backend_object_close (JBackend*, gpointer);
+gboolean j_backend_object_status(JBackend*, gpointer, gint64*, guint64*);
+gboolean j_backend_object_sync(JBackend*, gpointer);
 
-gboolean j_backend_object_status (JBackend*, gpointer, gint64*, guint64*);
-gboolean j_backend_object_sync (JBackend*, gpointer);
+gboolean j_backend_object_read(JBackend*, gpointer, gpointer, guint64, guint64, guint64*);
+gboolean j_backend_object_write(JBackend*, gpointer, gconstpointer, guint64, guint64, guint64*);
 
-gboolean j_backend_object_read (JBackend*, gpointer, gpointer, guint64, guint64, guint64*);
-gboolean j_backend_object_write (JBackend*, gpointer, gconstpointer, guint64, guint64, guint64*);
+//kv backend ->
+gboolean j_backend_kv_init(JBackend*, gchar const*);
+void j_backend_kv_fini(JBackend*);
 
-gboolean j_backend_kv_init (JBackend*, gchar const*);
-void j_backend_kv_fini (JBackend*);
+gboolean j_backend_kv_batch_start(JBackend*, gchar const*, JSemanticsSafety, gpointer*);
+gboolean j_backend_kv_batch_execute(JBackend*, gpointer);
 
-gboolean j_backend_kv_batch_start (JBackend*, gchar const*, JSemanticsSafety, gpointer*);
-gboolean j_backend_kv_batch_execute (JBackend*, gpointer);
+gboolean j_backend_kv_put(JBackend*, gpointer, gchar const*, gconstpointer, guint32);
+gboolean j_backend_kv_delete(JBackend*, gpointer, gchar const*);
+gboolean j_backend_kv_get(JBackend*, gpointer, gchar const*, gpointer*, guint32*);
 
-gboolean j_backend_kv_put (JBackend*, gpointer, gchar const*, gconstpointer, guint32);
-gboolean j_backend_kv_delete (JBackend*, gpointer, gchar const*);
-gboolean j_backend_kv_get (JBackend*, gpointer, gchar const*, gpointer*, guint32*);
+gboolean j_backend_kv_get_all(JBackend*, gchar const*, gpointer*);
+gboolean j_backend_kv_get_by_prefix(JBackend*, gchar const*, gchar const*, gpointer*);
+gboolean j_backend_kv_iterate(JBackend*, gpointer, gchar const**, gconstpointer*, guint32*);
 
-gboolean j_backend_kv_get_all (JBackend*, gchar const*, gpointer*);
-gboolean j_backend_kv_get_by_prefix (JBackend*, gchar const*, gchar const*, gpointer*);
-gboolean j_backend_kv_iterate (JBackend*, gpointer, gchar const**, gconstpointer*, guint32*);
```

Comment:
See above, this contains a lot of unrelated formatting changes.

Reply:
I'll remove those space only changes